.gitignore

```diff
@@ -30,5 +30,4 @@ node_modules/
 
 # Build-related directories
 docs/api/
-es5/
```

Comment:
mmmm i removed this commit.
i just need this in my fork to be able to add my git fork as a npm depedency